### ISSUE TITLE
RHEL 7 with php7

### ISSUE THREAD

**Issue by pozzo-balbi** [drlippman/IMathAS#54]
Hi, I needed to install php 5.4 to get IMathAS to run, since php 7 does not have php-mysql but only php-mysqli or php-mysqlnd. Not sure if MySQL/MariaDB running with utf8mb4 would work. I created the database with default latin1 characterset and it worked. Since php 7 does offer improved speed, something to consider for further releases.

**Comment by drlippman** [comment]
I am in the process of a full code rewrite to use PDO instead of php-mysql.  It should be released in the next month or so.
On Sep 8, 2016 12:53 PM, pozzo-balbi <notifications@github.com> wrote:Hi, I needed to install php 5.4 to get IMathAS to run, since php 7 does not have php-mysql but only php-mysqli or php-mysqlnd. Not sure if MySQL/MariaDB running with utf8mb4 would work. I created the database with default latin1 characterset and it worked. Since php 7 does offer improved speed, something to consider for further releases.
